spacy/tests/test_cli.py

```diff
@@ -32,6 +32,91 @@ def test_cli_converters_conllu2json():
     assert [t["ner"] for t in tokens] == ["O", "B-PER", "L-PER", "O"]
 
 
+def test_cli_converters_iob2json():
+    lines = [
+        "I|O like|O London|I-GPE and|O New|B-GPE York|I-GPE City|I-GPE .|O",
+        "I|O like|O London|B-GPE and|O New|B-GPE York|I-GPE City|I-GPE .|O",
+        "I|PRP|O like|VBP|O London|NNP|I-GPE and|CC|O New|NNP|B-GPE York|NNP|I-GPE City|NNP|I-GPE .|.|O",
+        "I|PRP|O like|VBP|O London|NNP|B-GPE and|CC|O New|NNP|B-GPE York|NNP|I-GPE City|NNP|I-GPE .|.|O",
+    ]
+    input_data = "\n".join(lines)
+    converted = iob2json(input_data, n_sents=10)
+    assert len(converted) == 1
+    assert converted[0]["id"] == 0
+    assert len(converted[0]["paragraphs"]) == 1
+    assert len(converted[0]["paragraphs"][0]["sentences"]) == 4
+    for i in range(0, 4):
+        sent = converted[0]["paragraphs"][0]["sentences"][i]
+        assert len(sent["tokens"]) == 8
+        tokens = sent["tokens"]
+        assert [t["orth"] for t in tokens] == ["I", "like", "London", "and", "New", "York", "City", "."]
+        assert [t["ner"] for t in tokens] == ["O", "O", "U-GPE", "O", "B-GPE", "I-GPE", "L-GPE", "O"]
+
+
+def test_cli_converters_conll_ner2json():
+    lines = [
+        "-DOCSTART- -X- O O",
+        "",
+        "I	O",
+        "like	O",
```

Comment:
Small suggestion for futureproofing the test: Maybe we should use `\t` here to make the tabs more visible/explicit?

Reply:
That would be fine. Or should we move the tests to external files?

Reply:
I think in this case, it's short enough to have it in the same file. It makes the test a lot easier to read and to follow if it's all there.

Reply:
What about having examples to include in the docs? Would you want separate ones?